ci/env/install-dependencies.sh

```diff
@@ -316,11 +312,18 @@ install_pip_packages() {
     if [ "$status" != "0" ]; then
       echo "${status}" && return 1
     fi
-  fi
 
-  # Default requirements
-  if [ "${MINIMAL_INSTALL-}" != 1 ]; then
-    pip install -r "${WORKSPACE_DIR}"/python/requirements/requirements_default.txt
+    # Repeat for requirements_test.txt
+    local status="0";
+    local errmsg="";
+    for _ in {1..3}; do
+      errmsg=$(CC=gcc pip install -c "${WORKSPACE_DIR}"/python/requirements.txt -r "${WORKSPACE_DIR}"/python/requirements_test.txt 2>&1) && break;
```

Comment:
`requirements.txt` may not be necessary- it's a duplicate of `ray[all]` anyways.

Reply:
Let's keep it as a constraints file (otherwise sub-dependencies of pinned dependencies may update versions to incompatible ones)